src/cwCompassImporter.cpp

```diff
@@ -53,7 +53,7 @@ void cwCompassImporter::runTask()
         parseFile();
 
         //Fix invalid station names
-        StationRenamer.renameInvalidStations();
+//        StationRenamer.renameInvalidStations();
```

Comment:
Actually ironically I think it wouldn't have caused a merge conflict since I didn't touch that line, but with both PRs touching that line it probably would cause a merge conflict.  Let's just wait until we've merged the other PR